GameOfLife.java

```diff
@@ -84,31 +109,62 @@ private static int[][] evolve(int[][] board) {
 	// Assumes that i is at least 1 and at most the number of rows in the board - 1. 
 	// Assumes that j is at least 1 and at most the number of columns in the board - 1. 
 	// Uses the count(board,i,j) function to count the number of alive neighbors.
-	private static int cellValue(int[][] board, int i, int j) {
-		//// Replace the following statement with your code.
-		return 0;
+	public static int cellValue(int[][] board, int i, int j) {
+		int cellValue = board[i][j];
+		int living = count(board, i, j);
+		if(cellValue == 1){
+			if (living < 2){
+				cellValue = 0;
+			}
+			if (living == 3 || living == 2){
+				cellValue = 1;
+			}
+			if (living > 3){
+				cellValue = 0;
+			}
```

Comment:
When you chain `if`s with else it skip them when it already know there's no change to get into their scopes.
```suggestion
			if (living < 2){
				cellValue = 0;
			}
			else if (living == 3 || living == 2){
				cellValue = 1;
			}
			else {
				cellValue = 0;
			}
```